docs/03-concepts/13-grafana-helm-setup.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: Grafana Helm Setup
+permalink: /docs/concepts/grafana-helm-setup
+---
+
+# Grafana Helm Setup
+
+<details>
+<summary><h2>Introduction</h2></summary>
+
+This guide explains how to set up Grafana for monitoring Cadence workflows and services using Helm charts. Helm simplifies the deployment and management of Grafana in Kubernetes environments. Pre-configured dashboards for Cadence are available to visualize metrics effectively.
+
+</details>
+
+<details>
+<summary><h2>Prerequisites</h2></summary>
+
+Before proceeding, ensure the following:
+
+- Kubernetes cluster is up and running.
+- Helm is installed on your system. Refer to the [Helm installation guide](https://helm.sh/docs/intro/install/).
+- Access to the Cadence Helm charts repository.
+
+</details>
+
+<details>
+<summary><h2>Setup Steps</h2></summary>
+
+### Step 1: Create a Namespace (Optional, but recommended)
+
+```bash
+kubectl create namespace <namespace>
+```
```

Comment:
This step is not required. the helm install commands below has `--create-namespace`